spec/services/catalog/provider_control_parameters_spec.rb

```diff
@@ -7,27 +7,43 @@
   let(:params) { portfolio_item.id }
   let(:provider_control_parameters) { described_class.new(params) }
   let(:portfolio_item) { create(:portfolio_item, :service_offering_source_ref => source_id) }
-  let(:project1) do
-    TopologicalInventoryApiClient::ContainerProject.new('name'      => "project one",
+  let(:project1_name) { 'project-one' }
+  let(:project2_name) { 'project-two' }
+  let(:project_default_name) { 'default' }
+  let(:project_default) do
+    TopologicalInventoryApiClient::ContainerProject.new('name'      => project_default_name,
                                                         'source_id' => "1")
   end
-  let(:project2) do
-    TopologicalInventoryApiClient::ContainerProject.new('name'      => "project one",
```

Comment:
Note, `project2` previously had the same name as `project1` so the test to check that the first element was `project one` would not have caught this any sorting errors.